### ISSUE TITLE
Plugin enhancement 

### ISSUE THREAD

**Issue by Krishna-art-dev** [SaurabhKaperwan/CSX#63]
Is there any way to add tags like clean audio, unofficial dub?

Recent big change in how plugin works, removed details in movie title that is provided by the site 

Like 
![Screenshot_2024-09-15-16-01-48-71_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/user-attachments/assets/5cb4b588-4649-4dc2-9231-1c25a8b3268d)
![Screenshot_2024-09-15-16-01-59-14_cfa2fd239c8cded9285432912d0e0818](https://github.com/user-attachments/assets/75e02f70-27ce-47bd-8fa6-fe4791fe5349)
![Screenshot_2024-09-15-16-02-17-03_cfa2fd239c8cded9285432912d0e0818](https://github.com/user-attachments/assets/b87b439f-f9af-4ac3-9d9b-6d974eded4eb)
I am talking about this, moviesmod didn't have Hindi dub, and there is no way to know that because of recent change, had to go to site to know ( search tag only show half title as title is long)

Recently watching "double ismart" full4movie had clean audio, rogmovies also have clean audio, top movies didn't, there isn't any way to know that as going inside movie remove title ( I had less problem in this movie because)
![Screenshot_2024-09-15-16-14-24-03_cfa2fd239c8cded9285432912d0e0818](https://github.com/user-attachments/assets/17c9b25a-133d-44bf-9aad-9888e34c8090)
You can do like rowdy did showing movie is in web-dl, bluray , on movie thumbnail 

Thanks
Some extension doesn't work please fix that

**Comment by Krishna-art-dev** [comment]
![Screenshot_2024-09-15-16-19-16-43_cfa2fd239c8cded9285432912d0e0818](https://github.com/user-attachments/assets/7f971e50-2a1d-415f-9ebc-db3adaa58409)
Like this,

**Comment by SaurabhKaperwan** [comment]
I did this because shorter titles looks good

**Comment by SaurabhKaperwan** [comment]
And I can't mimic how rowdy work

**Comment by SaurabhKaperwan** [comment]
It is either short title or full title

**Comment by SaurabhKaperwan** [comment]
Or I can add  long title in the description

**Comment by SaurabhKaperwan** [comment]
Which extensions are not working?

**Comment by Krishna-art-dev** [comment]
> I did this because shorter titles looks good

Yes I like that also, that's why am saying add tags like that pic

Hmm you can't able to add tags like that

That's good add that long title in description, that will be good

**Comment by Krishna-art-dev** [comment]
> Which extensions are not working?

Should I make issue for different title one by one, or all in one?

**Comment by SaurabhKaperwan** [comment]
> > Which extensions are not working?
> 
> Should I make issue for different title one by one, or all in one? 

Make different issue

**Comment by Krishna-art-dev** [comment]
Ok

> > > Which extensions are not working?
> > 
> > 
> > Should I make issue for different title one by one, or all in one?
> 
> Make different issue

Ok

**Comment by SaurabhKaperwan** [comment]
I'm going to revert back to the old title style. No adding in description

**Comment by Krishna-art-dev** [comment]
> I'm going to revert back to the old title style. No adding in description

Ok, do as you think will be the best

**Comment by Zafor7869** [comment]
![Screenshot_20240920_155824](https://github.com/user-attachments/assets/4b1d08ef-33ee-4e86-925b-b267f6c2e8e7)

Bro I want to know that how to work of this extractor, could you please explain little bit.

**Comment by SaurabhKaperwan** [comment]
Make changes in vega to show cam rips in posters

**Comment by Krishna-art-dev** [comment]
> Made changes in vega to show cam rips in posters

Thanks
